components/task-management-hub.tsx

```
@@ -0,0 +1,279 @@
+			assignee: "Jane Smith",
+			labels: ["feature"],
+			createdAt: new Date(),
+			updatedAt: new Date(),
+		},
+	]);
+
+	const handleVoiceRecording = useCallback(async (audioBlob: Blob) => {
+		console.log("Voice recording completed:", audioBlob);
+		// Here you would send the audio to the voice API
+		setShowVoiceRecorder(false);
+	}, []);
```

Biome (2.1.2)

[error] This async function lacks an await expression.

Remove this async modifier, or add an await expression in the function.

Async functions without await expressions may not need to be declared async.

(lint/suspicious/useAwait)